core/src/main/java/uk/ac/bbsrc/tgac/miso/core/service/printing/LabelCanvas.java

```diff
@@ -90,18 +90,24 @@ public final void multilineText(double x, double y, double height, int numCharsP
     lines//
         .filter(input -> !LimsUtils.isStringBlankOrNull(input.getValue()))//
         .reduce(0, (offset, input) -> {
+          // This block is full. This line of text gets ignored. So sad.
+          if (offset >= maxLines) {
+            return offset;
+          }
           int line;
-          for (line = offset; line < maxLines - 1 && input.getValue().length() > line * numCharsPerLine; line++) {
-            text(x + direction.xMultiplier * 1.1 * height * line, y + direction.yMultiplier * 1.1 * height * line, height, direction,
+          // Print this text over as many lines as it needs and or the block can hold
+          for (line = 0; line + offset < maxLines && input.getValue().length() > line * numCharsPerLine; line++) {
+            String str;
+            // If we are out of lines and this string needs more than one, than add some dots.
```

Comment:
`If we are on our last line and this string needs more than one, then add some dots at the end of this line to stand in for the additional line(s).`